ansible/roles/common/templates/settings.ini.j2

```diff
@@ -0,0 +1,64 @@
+[Program]
```

Comment:
This file should be renamed to be `meteordemod_settings.ini.j2` and the ansible bit updated to use that new local filename (without changing the "remote" one where it gets installed to).